tests/app-pip/requirements.txt

```diff
@@ -0,0 +1 @@
+urllib3==1.26.2
```

Comment:
Using `~=` could save you a dependabot PR every now and then.

Reply:
Fixed!